src/Simulation/Native/src/external/fusion.hpp

```diff
@@ -58,21 +65,58 @@ class Fusion{
 		handle_controls(empty_matrix, empty_vec, {}); // remove all current control qubits (this is a GLOBAL factor)
 	}
 	
+	IndexSet get_target_set() const& {
```

Comment:
same comment as in fused.hpp: this might be equivalent to `const IndexSet& get_target_set()` declaration but it's more traditional to do it with type modifiers unfront. Also make function const: `const IndexSet& get_target_set() const`